web/src/lib/components/admin-page/settings/storage-template/supported-datetime-panel.svelte

```diff
@@ -1,11 +1,14 @@
 <script lang="ts">
+  import { locale } from '$lib/stores/preferences.store';
   import type { SystemConfigTemplateStorageOptionDto } from '@immich/sdk';
   import * as luxon from 'luxon';
 
   export let options: SystemConfigTemplateStorageOptionDto;
 
   const getLuxonExample = (format: string) => {
-    return luxon.DateTime.fromISO(new Date('2022-09-04T20:03:05.250').toISOString()).toFormat(format);
+    return luxon.DateTime.fromISO(new Date('2022-09-04T20:03:05.250').toISOString(), { locale: $locale }).toFormat(
```

Comment:
All this does is add a `Z` lol. Maybe just use `fromISO('2022-09-05T00:03:05.250Z')` instead?